NEWS.md

```diff
@@ -0,0 +1,17 @@
+This file contains a summary of important user-visible changes.
+
+ethos 0.1.0
+===========
+
+This is the initial release of the Ethos proof checker.  Ethos implements the Eunoia logical framework which is a logical framework targeted at SMT solvers.  It allows users to define proof formats and write proofs.
+
+This release of Ethos is associated with the 1.2.0 release of the SMT solver cvc5.  It can check the proofs generated by cvc5's native proof format `cpc`.  Ethos and Eunoia have reached a certain level of stability, but they are still under active development, and users should expect breaking changes in future releases.
```

Comment:
```suggestion
This release of Ethos is associated with the 1.2.0 release of the SMT solver cvc5.  It can check the proofs generated by cvc5's native proof format `cpc`.  Ethos and Eunoia have reached a certain level of stability, but they are still under active development.
```

Reply:
I think stating "under active development" is enough here. We don't want to give the impression that we will break changes regularly.